pyhelm/repo.py

```diff
@@ -102,7 +111,7 @@ def from_repo(repo_url, chart, version=None, headers=None):
         for url in metadata['urls']:
             fname = url.split('/')[-1]
             try:
-                fobj = cStringIO.StringIO(
+                fobj = StringIO.StringIO(
```

Comment:
`io.StringIO` (and `io.BytesIO`) is a class and not a module, so call it directly